pkg/services/alertmanager.go

```diff
@@ -33,13 +33,13 @@ type AlertmanagerNotification struct {
 
 // AlertmanagerOptions cluster configuration
 type AlertmanagerOptions struct {
-	Targets            []string   `json:"targets"`
-	Scheme             string     `json:"scheme"`
-	APIPath            string     `json:"apiPath"`
-	BasicAuth          *BasicAuth `json:"basicAuth"`
-	BearerToken        string     `json:"bearerToken"`
-	InsecureSkipVerify bool       `json:"insecureSkipVerify"`
-	Timeout            int        `json:"timeout"`
+	Targets     []string                       `json:"targets"`
+	Scheme      string                         `json:"scheme"`
+	APIPath     string                         `json:"apiPath"`
+	BasicAuth   *BasicAuth                     `json:"basicAuth"`
+	BearerToken string                         `json:"bearerToken"`
+	Transport   httputil.HTTPTransportSettings `json:"transport"`
```

Comment:
We cannot break backwards compatibility; `InsecureSkipVerify` needs to stick around.

We'll need to give some thought on how to preserve backwards compatibility for `InsecureSkipVerify`. One idea could be to define the other `httputil.HTTPTransportSettings` fields inline in each struct.